setup.py

```diff
@@ -30,6 +35,10 @@
       install_requires=['requests>=2.4.3,!=2.12.0,!=2.12.1', 'requests-toolbelt', 'boto>=2.9.7', 'pyyaml', 'six'],
       tests_require=tests_require,
       packages=find_packages(),
+      entry_points=ENTRY_POINTS,
+      extras_require = {
+        'testing':  ["pytest"],
```

Comment:
E121 continuation line under-indented for hanging indent
E241 multiple spaces after ':'